libs/analytics/angular-analytics/src/lib/cookie-consent/cookie-consent.component.ts

```diff
@@ -20,10 +20,10 @@ import {
 })
 export class CookieConsentComponent {
   public cookies: Cookie[] = [
-    new RequiredCookie(),
+    // new RequiredCookie(),
```

Comment:
are you sure we should comment out tracking and required.  our advisors suggested that we need to track usage ... does this remove this ability in the future ... maybe we should keep it for now.